Xwt.Gtk/Xwt.GtkBackend/ButtonBackend.cs

```diff
@@ -272,6 +307,15 @@ void HandleSizeAllocated (object o, Gtk.SizeAllocatedArgs args)
 			Widget.Child.SizeAllocate (args.Allocation);
 			args.RetVal = true;
 		}
+
+		protected override void Dispose (bool disposing)
+		{
+			if (labelWidget != null) {
+				labelWidget.Realized -= HandleStyleUpdate;
```

Comment:
`labelWidget` is created either in `SetFormattedText()` (events subscribed) or in `SetContent ()` (events not subscribed), so this will thrwo NRE for regular text (SetFormattedText() not called).